### DISCUSSION TITLE
Is DAB supports Azure Synapse dedicated sql pool?

### DISCUSSION THREAD

**Discussion by Madhan-Rommala** [Azure/data-api-builder#1515] [Q&A] [answered]
Hi Guys, I just wanted to understand that is DAB supports Azure Synapse dedicated sql pool? can we created API endpoints on Azure Synapse DB's?

**Comment by Aniruddh25** [comment, marked as answer]
Hi @Madhan-Rommala,
DAB does not support dedicated sql pool yet since as of now we rely on the `FOR JSON PATH` clause to format the response as JSON for Azure SQL DB as per the docs here: https://learn.microsoft.com/en-us/sql/relational-databases/json/format-query-results-as-json-with-for-json-sql-server?view=azuresqldb-current 

Thanks
Aniruddh.

**Reply by himanshufreeman** [reply]
Does it support synapse on-demand sql pool? 

I had limited success with dedicated sql pool using sqldw as the database type and defining entity primary keys but got stuck on the 'STRING_ESCAPE' is not a recognized built-in function name.

**Comment by Narming1868** [comment]
Hi, is this also causing the error message?
 {
    "error": {
        "code": "DatabaseOperationFailed",
        "message": "'FOR' clause is not supported in this version of Synapse SQL.",
        "status": 500
    }
}

Thanks for your help.
